config/routes.rb

```diff
@@ -0,0 +1,16 @@
+Rails.application.routes.draw do
+  get 'splash/index'
+  get 'foods/index'
+  devise_for :users, controllers: {
+    sessions: 'users/sessions',
+    registrations: 'users/registrations'
+  }
+  resources :users
+  resources :splash
+
+  resources :groups do
+    resources :foods
```

Comment:
- [ ] Nice work defining the food routes :clap: However it would be nice to limit to only the **new** and **create** routes since those are the only two routes that we need for the foods.
Example:
```rb
 resources :groups do
    resources :foods, only: %i[new create]
 end
```

Reply:
Thanks @LionRouge1, the changes asked by you has been done. 😃